### PR TITLE
Bastion vm fixture

### PR DESCRIPTION
**Pull request by anniehedgpeth**
## Background

[Asana](https://app.asana.com/0/1181500399442529/1201469300087917/f)
This branch adds a fixture module for the creation of a bastion virtual machine. More details in the Asana task above.

## How Has This Been Tested

I tested the private-active-active test locally, but I will test again here and also test private-tcp-active-active, as they are the two that require bastion virtual machines.

## This PR makes me feel

reusable

### CODE REVIEW

File: fixtures/bastion_vm/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# FIXTURE: TFE Bastion Virtual Machine Module
+
+While the root module of this repository gives you the option to create an [Azure Bastion](../modules/bastion/README.md) (as this is the recommended method for accessing your TFE instance), there may be times for development environments when you will need SSH access to the bastion host (which the Azure Bastion does not provide). This module will create a virtual machine that you will be able to use as a bastion host for your TFE instance.
```
Comment (aaron-lane) [inline fixtures/bastion_vm/README.md:3]: This line should be split up for improved gitability 🪘

**Review by aaron-lane — CHANGES_REQUESTED** [review]
Nice work, surpassed only by the amazing feat of the tests running successfully on the first try 😮 

Beyond the one request below, we also need to drop [private-tcp-active-active/proxy.tf](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise/blob/1df02291ef8485a213a3cb0b58333ee66d95fb1e/tests/private-tcp-active-active/proxy.tf).

**Comment by anniehedgpeth** [comment]
> Nice work, surpassed only by the amazing feat of the tests running successfully on the first try 😮
> 
> Beyond the one request below, we also need to drop [private-tcp-active-active/proxy.tf](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise/blob/1df02291ef8485a213a3cb0b58333ee66d95fb1e/tests/private-tcp-active-active/proxy.tf).

Ha, oh no, I deleted the looooong list of failed tests because I was tired of scrolling. 😆

**Comment by anniehedgpeth** [comment]
> 



> > Nice work, surpassed only by the amazing feat of the tests running successfully on the first try 😮
> > Beyond the one request below, we also need to drop [private-tcp-active-active/proxy.tf](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise/blob/1df02291ef8485a213a3cb0b58333ee66d95fb1e/tests/private-tcp-active-active/proxy.tf).
> 
> Ha, oh no, I deleted the looooong list of failed tests because I was tired of scrolling. 😆

Oh wait, wrong PR, yeah, this one was easy and only once.

**Review by aaron-lane — APPROVED** [review]
Sorry, I was waiting for `proxy.tf` to be removed when it's completely irrelevant to this work 😅

**Pull request merged**